docs/reference/pulumi/pulumi-cloud.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Using Pulumi Cloud as a Backend
+description: Use pulumi cloud as a state backend when deploying your nitric applications
+---
+
+Nitric uses the [Pulumi Automation API](https://www.pulumi.com/automation/) for it's out of the box provider plugins, this can be a bit confusing at first as there is no need to create a workspace along with your project to manage your stacks. These stacks are instead managed by the program that uses the automation API to interact with Pulumi to get your infrastructure deployed, this could come in the form of a CLI, API or even plugins to other software.
```

Comment:
```suggestion
Nitric uses the [Pulumi Automation API](https://www.pulumi.com/automation/) for its pre-built provider plugins. At first, this may cause some confusion as there is no requirement to create a workspace for managing stacks alongside your project. Instead, the program that interacts with Pulumi via the automation API handles the management of these stacks, which could take the form of a CLI, API, or plugins for other software.
```